src/components/scenes/Loans/LoanCreateConfirmationScene.js

```diff
@@ -126,7 +127,9 @@ export const LoanCreateConfirmationScene = (props: Props) => {
   if (loanAccountError != null) return <Alert title={s.strings.error_unexpected_title} type="error" message={translateError(loanAccountError)} />
 
   return loanAccount == null ? (
-    <FillLoader />
+    <SceneWrapper background="theme">
+      <FillLoader />
+    </SceneWrapper>
```

Comment:
**Optional change:**
It looks cleaner and is more consistent to have an early exit if statement for the loading/error render cases before the default rendering case:

```js
if (loanAccountError != null) return <Alert ... />

if (loanAccount == null) 
return (
  <SceneWrapper background="theme">
    <FillLoader />
  </SceneWrapper>
)

return (
  <FormScene...>
```